prime-router/settings/prod/0058-update-clark-county-oh-doh .yml

```diff
@@ -0,0 +1,26 @@
+# ./prime multiple-settings set --env prod --input ./settings/prod/0058-setup-clark-county-oh.yml
+---
+- name: oh-ccchd-doh
+  description: Ohio Department of Health - Clark County
+  jurisdiction: COUNTY
+  stateCode: OH
+  countyName: Clark
+  receivers:
+    - name: elr-download
+      organizationName: oh-ccchd-doh
+      topic: covid-19
+      customerStatus: testing
+      jurisdictionalFilter: [ "matches(ordering_facility_state, OH)", "matches(ordering_facility_county, Clark)" ]
```

Comment:
With Jim's new filter set up I think you should move the jurisdictional filter up to the organization level rather than at the receiver level. Same for the staging yml

Reply:
Great catch.